src/test/java/data/CameraShotTest.java

```diff
@@ -11,22 +11,24 @@
 public class CameraShotTest {
     CameraShot cameraShot;
     @Before
-    public void initialize() {
+    public void initializeTest() {
         cameraShot = new CameraShot("camerashot-1");
```

Comment:
The test for the initialize method could be added. An constructor method is simply tested by calling the constructor and verifying that no exception is thrown.